feeds/scheduler/feed_group.go

```diff
@@ -38,10 +38,9 @@ func (fg *FeedGroup) Run() {
 }
 
 func (fg *FeedGroup) PollAndPublish() (int, error) {
-	pkgs, errs := fg.poll()
-	if len(errs) > 0 {
-		return 0, errs[0]
-	} else if len(pkgs) == 0 {
+	// poll() logs errs, so no need to handle.
```

Comment:
if `poll()` handles errors then they shouldn't be passed down. Not directly introduced by this PR but we could fix this whilst reworking :).

Reply:
I think that removing the tests (or this check) is fine if we're removing this pass down of errors. Tests should verify the original feeds output the errors which is handled on a feed by feed basis. We could introduce a parameterized `Logger` throughout which would allow us to provide a [NewNullLogger()](https://github.com/sirupsen/logrus/blob/master/hooks/test/test.go#L42) as the `Logger` for use within the `FeedGroup`, but I think this is unnecessarily complex for the current test scenario.